packages/react-core/src/filters/FilterTypes.d.ts

```diff
@@ -0,0 +1,16 @@
+export enum FilterTypes {
+  IN = 'in',
+  BETWEEN = 'between', // [a, b] both are included
+  CLOSED_OPEN = 'closed_open', // [a, b) a is included, b is not
+  TIME = 'time',
+  STRING_SEARCH = 'stringSearch'
+}
+
+export const filterFunctions: Record<
```

Comment:
UppperCamelCase for types please

Reply:
But... it's not a type. It's a constant.

Reply:
But that would be a breaking change. With this line 9, what I did is typing the exported map object `filterFunctions` that's used internally in Filter.js. That's all.

Reply:
Sorry for original comment, misread. 
Agree with @Clebal that changing enum names would be unintended breaking change.